packages/dds/merge-tree/api-extractor-esm.json

```diff
@@ -0,0 +1,12 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../common/build/build-common/api-extractor-base-esm.json",
+	"messages": {
+		"extractorMessageReporting": {
+			// The ESM types complain about missing exports; I don't understand why CJS doesn't
+			"ae-forgotten-export": {
```

Comment:
Is there a work item to follow up on?

Reply:
The difference could be the use of `--local`. `api-extractor` says `--local` suppresses some checks. Not `--local` is good but slower.

Reply:
--local has no effect on the errors. Without the suppression, the errors reported are:

Error: src/localReference.ts:241:2 - (ae-forgotten-export) The symbol "IRefsAtOffset" needs to be exported by the entry point index.d.mts
Error: src/mergeTreeNodes.ts:402:2 - (ae-forgotten-export) The symbol "ISegmentLeaf" needs to be exported by the entry
point index.d.mts

I updated the comment with these details. AB#6647 is the followup.